File: 001_class.js
class Person {
    constructor(name, age, gender) {
        this.name = name;
        this.age = age;
        this.gender = gender;
    }

    introduce() {
        console.log(`Hi, I'm ${this.name}, ${this.age} years old, and I am ${this.gender}.`);
    }
}

const person1 = new Person("Alice", 30, "female");
const person2 = new Person("Bob", 25, "male");

console.log(person1.name); // Output: Alice
person2.introduce(); // Output: Hi, I'm Bob, 25 years old, and I am male.

File: 001_class.cs
using System;

class Person
{
    public string Name { get; set; }
    public int Age { get; set; }
    public string Gender { get; set; }

    public Person(string name, int age, string gender)
    {
        Name = name;
        Age = age;
        Gender = gender;
    }

    public void Introduce()
    {
        Console.WriteLine($"Hi, I'm {Name}, {Age} years old, and I am {Gender}.");
    }
}

class Program
{
    static void Main()
    {
        Person person1 = new Person("Alice", 30, "female");
        Person person2 = new Person("Bob", 25, "male");

        Console.WriteLine(person1.Name); // Output: Alice
        person2.Introduce(); // Output: Hi, I'm Bob, 25 years old, and I am male.
    }
}


